src/main/java/com/softserveinc/ita/pageobjects/admin/SpecialitiesPage.java

```diff
@@ -1,27 +1,35 @@
 package com.softserveinc.ita.pageobjects.admin;
 
+import com.codeborne.selenide.SelenideElement;
+import com.softserveinc.ita.pageobjects.models.SpecialityEntity;
 import io.qameta.allure.Step;
 
 import static com.codeborne.selenide.Condition.appear;
 import static com.codeborne.selenide.Condition.disappear;
+import static com.codeborne.selenide.Condition.exactText;
+import static com.codeborne.selenide.Selenide.$$x;
 import static com.codeborne.selenide.Selenide.$x;
 
 public class SpecialitiesPage extends MainMenu {
 
+    private SelenideElement buttonNavigationLast = $x("//button[contains(@class,'paginator-navigation-last')]");
```

Comment:
create NAVIGATION_BUTTON_PATH_TEMPLATE: "//button[contains(@class,'paginator-navigation-%s')]"

Reply:
and use format(NAVIGATION_BUTTON_PATH_TEMPLATE, "last")
format(NAVIGATION_BUTTON_PATH_TEMPLATE, "first")
format(NAVIGATION_BUTTON_PATH_TEMPLATE, "next")